### PR TITLE
Add script for uploading whatsapp templates from csv

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

File: scripts/upload_whatsapp_templates/upload_whatsapp_templates.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+from csv import DictReader
+from urllib.parse import urljoin
+
+import attr
+import requests
+
+
+def length_limit(limit):
+    def validate_length(instance, attribute, value):
+        if len(value) > 1024:
```
Comment (erikh360) [inline scripts/upload_whatsapp_templates/upload_whatsapp_templates.py]: This should probably use the limit var

Comment (erikh360) [reply scripts/upload_whatsapp_templates/upload_whatsapp_templates.py]: I also see instance is not being used but it's probably the signature required by the validator.

**Review by erikh360 — COMMENTED** [review]
Just the one small comment but looks good otherwise

File: scripts/upload_whatsapp_templates/upload_whatsapp_templates.py (hunk 1 above)
Comment (rudigiesler) [reply scripts/upload_whatsapp_templates/upload_whatsapp_templates.py]: Whoops, good catch. Yeah it's required by the validator.

**Review by erikh360 — APPROVED** [review]
awesome 🍰

**Pull request merged**